plugins/modules/files/iso_customize.py

```diff
@@ -62,6 +62,11 @@
 '''
 
 EXAMPLES = r'''
+description:
+- In this example, the file /preseed/ubuntu.seed is deleted at first. Then we add local file
+- /path/to/ubuntu.seed to /path/to/ubuntu.seed inside ISO. So the file /path/to/ubuntu.seed
+- inside ISO is replaced.
+
```

Comment:
Please check other modules for examples on how to write this documentation. The structure is not right.

See: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block

Reply:
Thanks. change it as referring to module xml.py in community.general